### DISCUSSION TITLE
great job with the documentation

### DISCUSSION THREAD

**Discussion by joeromano** [pdbuzzboard/pdbuzzboard#14] [General]
Thanks for doing such a thorough job with the parts list and documentation. I was able to put one together in a few hours work. I test it out with my relative next week. 🤞 hoping that it helps a bit. Will report back with any results or changes.

For the mechanical assembly i found that a gardening glove had the right amount of rubberized coating on it such that all the components (vibration actuators, cables) were able to be super-glued to the outside nicely to secure them in place and prevent them from being damaged. It attenuates the vibration a little bit, but not that much being a thin glove, and makes a nice package for the user.

![8t2q40nAQ9Oy4i_PSq-HGg](https://github.com/pdbuzzboard/pdbuzzboard/assets/1622176/8b3d6964-917a-44aa-b7e8-f106f81ceed3)

**Comment by rebecca-sich** [comment]
This looks awesome, well done. Are you powering both the Arduino and the motor shield from one battery? Are you doing seperate controllers for each hand? 

Thanks!
